build.py

```diff
@@ -224,14 +224,44 @@ def upload_packages(packages, nightly=False):
         else:
             print "\t - Not uploading {}, already exists.".format(p)
     print ""
-    
-def run_tests():
+
+def run_tests(race, parallel, timeout):
     get_command = "go get -d -t ./..."
     print "Retrieving Go dependencies...",
+    sys.stdout.flush()
     run(get_command)
     print "done."
-    print "Running tests..."
-    code = os.system("go test ./...")
+    print "Running tests:"
+    print "\tRace: ", race
+    if parallel is not None:
+        print "\tParallel:", parallel
+    if timeout is not None:
+        print "\tTimeout:", timeout
+    sys.stdout.flush()
+    p = subprocess.Popen(["go", "fmt", "./..."], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    out, err = p.communicate()
+    if len(out) > 0 or len(err) > 0:
+        print "Code not formatted. Please use 'go fmt ./...' to fix formatting errors."
+        print out
+        print err
+        return False
+# GO vet for 1.4 is not working for me
```

Comment:
According to Go 1.4  there are lots of issues with our code base. And the command `go  tool vet -composites=false` doesn't ignore composites. Does anyone see what I am doing wrong here?

If I use Go 1.5 with the same command everything works as expected